pkg/clusterversion/cockroach_versions.go

```diff
@@ -301,6 +301,10 @@ const (
 	// ChangefeedIdleness is the version where changefeed aggregators forward
 	// idleness-related information alnog with resolved spans to the frontier
 	ChangefeedIdleness
+	// BackupDoesNotOverwriteLatestAndCheckpoint stops latest and checkpoint
```

Comment:
nit: `BackupDoesNotOverwriteLatestAndCheckpoint`is the version where we stop overwriting the LATEST and checkpoint files during backup execution. Instead, it writes new files alongside the old in reserved subdirectories.